src/Misc/baseError.ts

```diff
@@ -1,12 +1,46 @@
+import { ErrorCodesString } from "./errorCodes";
+
 /**
+ * BabylonJS error internal class wrapper. Extending Error for a class with docs
+ * fails typedoc validation, because lib.es5.d.ts does not have any docs for Error.
  * @ignore
- * Application error to support additional information when loading a file
  */
-export abstract class BaseError extends Error {
+class BabylonJSErrorWrapper extends Error {}

```

Comment:
typedoc-validate fails if the class extending error is not marked @ignore. Added an internal error wrapper class as workaround.

Reply:
I'm not sure I understand this. Error is an interface. Typedoc is complaining about the interface not having doc comments, but if we add a class inheritance layer then it works?

Also, isn't this going to introduce another prototype chain? Does that cause any issues?